releasenotes/notes/add-new-packageSigning-component-f1487f8d92f4e749.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+features:
+  - |
+    Creation of a new packageSigning component to collect Linux package signature information and improve signature rotation process.
```

Comment:
The public documentation is [on-going](https://github.com/DataDog/documentation/pull/20938). It will probably merged before this one so I'll update this with the full document, wdyt?